static/js/tools/map/info.tsx

```diff
@@ -39,6 +39,170 @@ export function Info(): JSX.Element {
+          <b>Median Age</b> across counties in{" "}
+          <a
+            href={
+              "#&sv=Median_Age_Person&pc=0&pd=country/USA&pn=United%20States%20of%20America&pt=Country&ept=County"
+            }
+          >
+            USA
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Median_Age_Person&pc=0&pd=geoId/06&pn=California&pt=State&ept=County"
+            }
+          >
+            California
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Median_Age_Person&pc=0&pd=geoId/12&pn=Florida&pt=State&ept=County"
+            }
+          >
+            Florida
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Median_Age_Person&pc=0&pd=geoId/36&pn=New%20York&pt=State&ept=County"
+            }
+          >
+            New York
+          </a>
+        </li>
+        <li>
+          <b>Median Income</b> across counties in{" "}
+          <a
+            href={
+              "#&sv=Median_Income_Person&pc=0&pd=country/USA&pn=United%20States&pt=Country&ept=County"
+            }
+          >
+            USA
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Median_Income_Person&pc=0&pd=geoId/36&pn=New%20York&pt=State&ept=County"
+            }
+          >
+            New York
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Median_Income_Person&pc=0&pd=geoId/48&pn=Texas&pt=State&ept=County"
+            }
+          >
+            Texas
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Median_Income_Person&pc=0&pd=geoId/53&pn=Washington&pt=State&ept=County"
+            }
+          >
+            Washington
+          </a>
+        </li>
+        <li>
+          <b>Attainment of Bachelor Degree or Higher</b> across counties in{" "}
+          <a
+            href={
+              "#&sv=Count_Person_EducationalAttainmentBachelorsDegreeOrHigher&pc=1&pd=country/USA&pn=United%20States%20of%20America&pt=Country&ept=County"
+            }
+          >
+            USA
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Count_Person_EducationalAttainmentBachelorsDegreeOrHigher&pc=1&pd=geoId/53&pn=Washington&pt=State&ept=County"
+            }
+          >
+            Washington
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Count_Person_EducationalAttainmentBachelorsDegreeOrHigher&pc=1&pd=geoId/17&pn=Illinois&pt=State&ept=County"
+            }
+          >
+            Illinois
+          </a>
+          ,{" "}
+          <a
+            href={
+              "#&sv=Count_Person_EducationalAttainmentBachelorsDegreeOrHigher&pc=1&pd=geoId/08&pn=Colorado&pt=State&ept=County"
+            }
+          >
+            Colorado
+          </a>
+        </li>
+        <li>
+          <b>Foreign Born</b> across counties in{" "}
```

Comment:
maybe we should skip this since it doesn't have a nicely formatted title.

i noticed these examples are mostly from demographics. how about UnemploymentRate?

Reply:
done